zthin-parts/zthin/bin/refresh_bootmap

```
@@ -59,6 +59,47 @@ function printCMDUsage {
   echo "${optionHelp}"
 } #printCMDUsage
 
+function cleanup_umountdir_and_disconnect_fcp {
+  : SOURCE: ${BASH_SOURCE}
+  : STACK:  ${FUNCNAME[@]}
+  # @Description:
+  #   Umount dir and cleanup all FCP devices.
+  umount $deviceMountPoint/proc
+  umount $deviceMountPoint/sys
+  umount $deviceMountPoint/run
+  umount $deviceMountPoint/dev
+  umount $deviceMountPoint
+  rm -rf $deviceMountPoint
+
+  cleanup_fcpdevices
+} # cleanup_umountdir_and_disconnect_fcp
+
+function cleanup_fcpdevices {
+  : SOURCE: ${BASH_SOURCE}
+  : STACK:  ${FUNCNAME[@]}
+  # @Description:
+  # Disconnect all FCPs
+  for i in "${!fcps[@]}"
```

ditto. ignore errors and print log msg when entering and exiting this function.